### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by Jolak5**
In this PR, I styled the app

### CODE REVIEW

File: src/pages/Home.jsx [hunk 1]
```diff
@@ -1,42 +1,69 @@
...
+                    strokeLinecap: 'butt',
+                    transition: 'stroke-dashoffset 0.5s ease 0s',
+                  },
+                  trail: {
+                    stroke: '#d6d6d6',
+                    strokeLinecap: 'butt',
+                  },
+                }}
+              />
+            </div>
```
Comment (ABDULALI3468) [inline src/pages/Home.jsx]: As it's a project requirement to use the same design and styles as used in the ZEPLIN template so please consider using the same set of styles. Currently, there you are using different colors please consider using the same color as in Zeplin's design.

## YOUR DESIGN:-

image


## ZEPLIN DESIGN:-

image

File: src/pages/Home.jsx [hunk 2]
```diff
@@ -1,42 +1,69 @@
...
+            <p className="booktitle">{book.author}</p>
             {' '}
             <div className="buttonDiv">
               {' '}
               <button type="button">Comments</button>
-              <button className="remove" type="button" onClick={() => { dispatch(deleteBook(book.item_id)); dispatch(deleteAPIBook(book.item_id)); }}>REMOVE</button>
+              <button className="remove" type="button" onClick={() => { dispatch(deleteBook(book.item_id)); dispatch(deleteAPIBook(book.item_id)); }}>Remove</button>
               <button type="button">Edit</button>
             </div>
           </div>
```
Comment (ABDULALI3468) [inline src/pages/Home.jsx]: As it's a project requirement to use the same design and styles as used in the ZEPLIN template so please consider using the same set of styles. In design there is a vertical line between the items kindly consider adding it as used in Zeplin Design.

## YOUR DESIGN:-

image



## ZEPLIN DESIGN:-

image

**Review by ABDULALI3468 — CHANGES_REQUESTED** [review]
# Changes Required 👍 

Hi @Jolak5  , 



Indeed you did an exceptional but still, there is room for changes to complete this project. 💯


## Highlights 
✅ Well-implemented design 🤷‍♀️
✅All linters are checked.
✅Project is professionally documented.
✅The right GitHub flow method has been used.
✅Professional pull request title and description.


 ## Required Changes ♻️

> **1-** Please consider using the same `font family AND other font styles` throughout the page as being used in the ZEPLIN design as its a project regiment.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification.

> * This is My **GitHub**: GitHub followers
> * This is My **Linkedin**: [Linkedin Badge](https://www.linkedin.com/in/abdul-ali-5400bb216/)&nbsp;
> * This is My **Twitter**: [Twitter Badge](https://twitter.com/mrabdul_ali)&nbsp;

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/pages/Home.jsx (hunk 2 above)
Comment (OyePriscilla) [reply src/pages/Home.jsx]: - [ ] As suggested earlier, please kindly implement the vertical line between these button elements according to the given design. 👍

File: src/pages/Home.jsx [hunk 3]
```diff
@@ -1,42 +1,69 @@
...
       { (books ?? []).filter((b) => b.item_id).map((book) => (
         <div className="eachBook" key={book.item_id} id={book.item_id}>
           <div className="left-pan">
             <h4>{book.category}</h4>
             {' '}
-            <h1>{book.author}</h1>
+            <h1>{book.title}</h1>
             {' '}
-            <p className="booktitle">{book.title}</p>
+            <p className="booktitle">{book.author}</p>
```
Comment (OyePriscilla) [inline src/pages/Home.jsx:28]: - [ ] Please kindly implement the appropriate font-family (`RobotoSlab`) for all the element highlighted in the screenshot below. ℹ️ Sincerely, we need to pay keen attention to details as developers.

## Your design
Untitled

## Previous reviewer comment 
Untitled

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @Jolak5 👏 👏 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

so-close-so-close-seth-meyers

 ## Highlights
Add styling to app ✔️
Made some required changes ✔️

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/pages/Home.jsx [hunk 4]
```diff
@@ -1,42 +1,71 @@
...
             {' '}
             <div className="buttonDiv">
               {' '}
               <button type="button">Comments</button>
-              <button className="remove" type="button" onClick={() => { dispatch(deleteBook(book.item_id)); dispatch(deleteAPIBook(book.item_id)); }}>REMOVE</button>
+              <span className="Line-2" />
+              <button className="remove" type="button" onClick={() => { dispatch(deleteBook(book.item_id)); dispatch(deleteAPIBook(book.item_id)); }}>Remove</button>
+              <span className="Line-2" />
               <button type="button">Edit</button>
             </div>
```
Comment (KanzaTahreem) [inline src/pages/Home.jsx:37]: - [ ] Good job adding the line between the buttons🥇🥇 However, please make sure that it is according to the given Zeplin design. Right now, I can see some white space in between and also it is not aligned according to the design guidelines. The width can be fixed by removing the `width` from `.Line-2`. See the images below for more clarity 💡

Your design 🔽 | Zeplin design 🔽
---| ---|
image | image

**Review by KanzaTahreem — CHANGES_REQUESTED** [review]
# Changes Required 🔁

Hi @Jolak5 👋

Good job so far! I know you worked hard to implement this project; I really appreciate the effort you put in! 
**_However, there is just one thing that need to be fixed. But you are almost there!_** 💪 🏁

so_close

## Highlights 🎯

There are some excellent points that you have made perfectly. Keep up the great work! 👏
- ✅ Well done following the given design
- ✅ Great job with CSS
- ✅ Descriptive PR and README file
- ✅ Correct Linters

However, there is always room for improvement.

## Required Changes 🛑

> _Please keep in mind that your long hours of work is appreciated. However, the code review process is for improving the code you wrote, and not for criticizing it. 😄_

_Please check the comments under the review._

## Optional suggestions ❕
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

N/A

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AnselemOdims — APPROVED** [review]
### Hi  @Jolak5 🙋‍♀️, 
Good job in making the required changes 👏 

# STATUS APPROVED ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Use the tag @AnselemOdims for a quicker reply


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**